index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+        integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+    <title>
+        Book Library
+    </title>
+</head>
+
+<body>
+
+
+    <div id="container">
+        <div id="head">
+            <div id="userContainer">
+                <div id="picture"></div>
+                <div id="name"></div>
+                <div id="location"></div>
+            </div>
+            <img src="personal_logo.jpeg" alt="personal_logo" id="personalLogo">
+        </div>
+        <div id="qoutes">
+            <div id="favoriteQoute">
+                Favourite qoute :
+            </div>
+            <div id="qoute">
+               
+            </div>
+            <div id="pokemon">
+               
+            </div>
+        </div>
+
+        <div id="aboutMe">
+            <div>
+                Aboute me:
+            </div>
+            <div id="aboutMeResponse">
+
+            </div>
+        </div>
+        <div id="friends">
+            
+
+        </div>
+        <div id="btns">
+            <button id="first">
+                Load User Data
+            </button>
+            <button id="second">
+                Display User
+            </button>
+        </div>
+    </div>
+    
```

Comment:
where is your hanlebars? you need to know and use it in the exam next week in order to pass